src/goto-programs/remove_const_function_pointers.cpp

```diff
@@ -12,15 +12,19 @@ Author: Thomas Kiley, thomas.kiley@diffblue.com
 #include "remove_const_function_pointers.h"
 
 #include <util/arith_tools.h>
+#include <util/format_expr.h>
 #include <util/simplify_expr.h>
 #include <util/std_expr.h>
 #include <util/symbol_table.h>
 
 #include "goto_functions.h"
 
 #define LOG(message, irep) \
-  debug() << "Case " << __LINE__ << " : " << message << "\n" \
-          << irep.pretty() << eom;
+  do { \
+    debug().source_location = irep.source_location(); \
+    debug() << message << ": " << format(irep) << eom; \
+  } \
+  while(0)
```

Comment:
Slight preference for `false` instead of `0` here; But we already use `0` in other places for this sort of construct so either is fine by me.